src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -909,7 +909,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     End If
                 Next
 
-                Return ImmutableArray.CreateRange(symbolSet)
+                Dim tmp = ImmutableArray.CreateRange(symbolSet)
```

Comment:
>tmp [](start = 20, length = 3)

Consider using more descriptive name. For example, "result".

Reply:
Done.